src/utils.js

```diff
@@ -18,7 +18,7 @@ if (isStyledComponent) {
     StyleSheet = secretInternals.StyleSheet
   }
 } else {
-  StyleSheet = require('styled-components/lib/models/StyleSheet').default
+  StyleSheet = require('styled-components/lib/models/StyleSheet').default // eslint-disable-line
```

Comment:
This gives you an error only when you have v4 installed, right?
I couldn't repro with v3.

If that's the case, this is safe to remove, as this PR is not adding v4 yet.